functions/lib/stripe/webhooks.py

```
@@ -513,6 +513,13 @@ def stripe_webhook_checkout_fulfilment(req: https_fn.Request) -> https_fn.Respon
     logger = Logger(f"stripe_webhook_logger_{uid}")
     logger.add_tag("uuid", uid)
 
+    # Handle GET requests (health checks from GCP/Firebase) - log warning and return early
+    if req.method == "GET":
+        logger.warning(
+            f"Received GET request to webhook endpoint. Returning 200 without processing."
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)